addon/noop-test-waiter.ts

```diff
@@ -1,4 +1,4 @@
-import { ITestWaiter, ITestWaiterDebugInfo } from './types';
+import { TestWaiter, TestWaiterDebugInfo } from './types';
```

Comment:
Same here, lets move this into `build-waiter.ts` and have that be the _only_ external importable thing

Reply:
Well thees are the types. I think we do want to support these types being exported, no?

Reply:
Wait, I misunderstood. Thought you were referring to exporting the types from the root. I agree with this.